src/jdk.jdeps/share/classes/com/sun/tools/javap/AttributeWriter.java

```diff
@@ -563,7 +564,7 @@ public void write(Attribute<?> a, CodeModel lr) {
                             printHeader(chop, "/* chop */");
                             indent(+1);
                             println("offset_delta = " + chop.offsetDelta());
-                            printMap("locals", chop.choppedLocals());
```

Comment:
Unfortunately I missed you added `printMap("locals", chop.choppedLocals());` and this is another addition to javap functionality.
This branch is not a place for wild extension of javap. 
Purpose of javap conversion from ASM to Classfile API is to keep "character-perfect" compatibility.
Please remove all non-standard additions.

Reply:
I'll remove it and file a bug that this information is missing.